### PR TITLE
feature/CU-bguca0_Create-an-UI-to-set-pins-count

### PR DESCRIPTION
**Pull request by YesLife**

### CODE REVIEW

**Comment by YesLife** [comment]
Task linked: [CU-bguca0 Create an UI to set pins count](https://app.clickup.com/t/bguca0)

**Comment by YesLife** [comment]
PC : 
image
Téléphone :
image

File: tests/unit/pins.spec.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
+import { mount } from '@vue/test-utils'
+import Vue from 'vue'
+import Pins from '../../src/views/Pins.vue'
+import VueMaterial from 'vue-material'
+
+/**
+ * Mount a mocked version of Turn component
```
Comment (Thurstag) [inline tests/unit/pins.spec.js:7]: `Mount a mocked version of Pins component`

File: tests/unit/pins.spec.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+ *
+ * @return {Wrapper<Turn>} Turn component
```
Comment (Thurstag) [inline tests/unit/pins.spec.js:9]: `@return {Wrapper<Pins>} Pins component`

**Review by Thurstag — CHANGES_REQUESTED** [review]

File: src/views/Pins.vue [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+<script>
+
+export default {
+  name: 'Pins',
+  data: () => ({
+    pinsCount: 1
+  }),
+  computed: {
+    getPins () { // Method to get the number of pins
```
Comment (Thurstag) [inline src/views/Pins.vue:72]: `pinsCount ()` pour le coverage de 100%

**Review by Thurstag — APPROVED** [review]

File: src/views/Pins.vue (hunk 3 above)
Comment (Thurstag) [inline src/views/Pins.vue:72]: `pinsCount ()` pour le coverage de 100%

**Review by Thurstag — CHANGES_REQUESTED** [review]

**Review by Tomansion — APPROVED** [review]

**Review by Thurstag — APPROVED** [review]

**Review by Fantorem — APPROVED** [review]

**Pull request merged**